### ISSUE TITLE
Header background colors override colors from config file.

### ISSUE THREAD

**Issue by enbock** [myyasuda/sphinx_materialdesign_theme#2]
Since `v0.1.4` seems that the color of the header bars are always blue.

My config is:
```python
html_theme_options = {
   *** snip ***

    # Customize css colors.
    # For details see link.
    # https://getmdl.io/customize/index.html
    #
    # Values: amber, blue, brown, cyan deep_orange, deep_purple, green, grey, indigo, light_blue,
    #         light_green, lime, orange, pink, purple, red, teal, yellow(Default: indigo)
    #'primary_color': 'light_green',
    'primary_color': 'green',
    # Values: Same as primary_color. (Default: pink)
    'accent_color': 'red',`

   *** snip ***
}
```

Looks like, that in the css class file for `.mdl-layout__header` loads in wrong order:
```html
<link rel="stylesheet" href="_static/material-icons.css" type="text/css" />
<link rel="stylesheet" href="_static/pygments.css" type="text/css" />
<link rel="stylesheet" href="_static/notosanscjkjp.css" type="text/css" />
<link rel="stylesheet" href="_static/roboto.css" type="text/css" />
<link rel="stylesheet" href="_static/material-design-lite-1.3.0/material.green-red.min.css" type="text/css" />
<link rel="stylesheet" href="_static/sphinx_materialdesign_theme.css" type="text/css" />
```

It should be in this order (so I belive):
```html
<link rel="stylesheet" href="_static/sphinx_materialdesign_theme.css" type="text/css" />
<link rel="stylesheet" href="_static/material-design-lite-1.3.0/material.green-red.min.css" type="text/css" />
```

I'm using the latest **Chrome** Browser on Windows 10.

**Comment by myyasuda** [comment]
This bug was caused by importing MDL's sass (_layout.scss) to related to header in theme's scss.
I removed `import material-design-lite/src/layout/layout` and uploaded fixed version (0.1.11) to PyPI.
Sorry, please build your document again with new version.

**Comment by enbock** [comment]
Hi myyasuda,

seems very nice now. Thank you for your quick reactions and fixes.
